<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkConvertToMultiLabelSegmentationWidgetControls</class>
 <widget class="QWidget" name="QmitkConvertToMultiLabelSegmentationWidgetControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>316</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Inputs:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QmitkMultiNodeSelectionWidget" name="inputNodesSelector" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupOutput">
     <property name="title">
      <string>Output:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioNewSeg">
        <property name="text">
         <string>Convert to new segmentation</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QmitkSingleNodeSelectionWidget" name="refNodeSelector" native="true"/>
        </item>
        <item>
         <widget class="QCheckBox" name="checkMultipleOutputs">
          <property name="toolTip">
           <string>If activated, the conversion will generate separate multi label segmentations for each input instead of joining all inputs in one multi label segmentation.</string>
          </property>
          <property name="text">
           <string>Convert inputs separately</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="radioAddToSeg">
        <property name="text">
         <string>Add to existing segmentation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkSingleNodeSelectionWidget" name="outputSegSelector" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupGrouping">
     <property name="title">
      <string>Grouping:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioMergeGroup">
        <property name="text">
         <string>Merge all inputs in new group</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioSingleGroup">
        <property name="text">
         <string>New group per input</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnConvert">
     <property name="text">
      <string>Convert</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>label</zorder>
  <zorder>inputNodesSelector</zorder>
  <zorder>groupGrouping</zorder>
  <zorder>btnConvert</zorder>
  <zorder>groupOutput</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkMultiNodeSelectionWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkMultiNodeSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkSingleNodeSelectionWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkSingleNodeSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
